### PR TITLE
Add ChainID to configuration

### PR DESCRIPTION
**Pull request by joaop21**
This PR adds the ChainID to configuration and retrieves it in the `/config` endpoint.

### CODE REVIEW

**Review by 0xrin1 — APPROVED** [review]
LGTM

File: .envrc.sample [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export ENV=local
+export CHAIN_ID=1
```
Comment (gabrielpoca) [inline .envrc.sample:1]: can we also get this on the `.envrc` please?

Comment (joaop21) [reply .envrc.sample:1]: It is already 👍

Comment (joaop21) [reply .envrc.sample:1]: It is already 👍

**Review by gabrielpoca — APPROVED** [review]

**Pull request merged**